docs/pages/operator/cluster-api-provider.mdx

```diff
@@ -0,0 +1,74 @@
+
+## Prerequisites 
+[clusterctl] CLI tool is used to install the vcluster provider and to generate a custom resource that will serve as a declarative definition of the cluster. It is possible to create the custom resource without clusterctl, but in this guide, we will be using it because it provides access to the templates for the `VCluster` custom resource. Please follow [the official installation instructions for clusterctl](https://cluster-api.sigs.k8s.io/user/quick-start.html#install-clusterctl).
+
+You will need cluster-admin permissions in a Kubernetes cluster where you wish to install the CAPI stack with the vcluster provider. This is required because vcluster instances that will be created by the vcluster provider may have arbitrary RBAC permission requirements. The vcluster provider installation with a more limited permission scope will be introduced in a future version.
+
+### Provider configuration - only clusterctl v1.1.4 and earlier
+If you are using clusterctl version 1.1.4 or earlier, you will need to add the vcluster provider to your local clusterctl configuration. To check which version you have installed look for the value of `GitVersion` field in the output of the `clusterctl version` command. You may skip to the next chapter if you are using a newer version.
+
+Open your local clusterctl configuration file `~/.cluster-api/clusterctl.yaml` and add vcluster to the list of provider as follows:
+
+```yaml
+providers:
+  - name: vcluster
+    url: https://github.com/loft-sh/cluster-api-provider-vcluster/releases/latest/infrastructure-components.yaml
+    type: InfrastructureProvider                                                           
+```
+
+Verify that the vcluster provider was added successfully using the `clusterctl config repositories` command, e.g.:
+```shell
+clusterctl config repositories | grep vcluster
+vcluster   InfrastructureProvider   https://github.com/loft-sh/cluster-api-provider-vcluster/releases/latest/                    infrastructure-components.yaml
+```
+
+## Installing the CAPI stack with the vcluster provider
+Switch your kubectl context to the cluster where you wish to install the CAPI stack with the vcluster provider and run the following command:
+```shell
+clusterctl init --infrastructure vcluster
+```
+
+The `"Your management cluster has been initialized successfully!"` message should eventually appear in the output.
+
+## Creating a vcluster
+We will use the clusterctl CLI tool to generate the Kubernetes resource definitions needed for vcluster creation by the CAPI provider. The output is in YAML format, and it can be saved into a file to be used later, or it can be forwarded in the command line into the kubectl command to immediately apply the resources in the cluster.  
+
+A vcluster instance created via vcluster provider is configured using clusterctl parameters and environment variables - CHART_NAME, CHART_REPO, CHART_VERSION, VCLUSTER_HOST, VCLUSTER_PORT, and HELM_VALUES.
```

Comment:
I'd change the format of the variables here to either a different format to differentiate them. If the variables are described somewhere else, I'd use option 1. If the variables don't have descriptions somewhere else, I'd use Option 2 or 3.

Option 1: modify text formatting
"A vcluster instance created via vcluster provider is configured using clusterctl parameters and environment variables: `CHART_NAME`, `CHART_REPO`, `CHART_VERSION`, `VCLUSTER_HOST`, `VCLUSTER_PORT`, and `HELM_VALUES`."

Option 2: table (also provides info about the variables)
| Variable | Description | Type | Default Value | Required/Optional |
| :---: | :--- | :---: | :---: | :---: |
| CHART_NAME | description of CHART_NAME | string | default value here | required |
| etc | etc | etc | etc | etc |

Option 3: plain list (also provides more info about the variables)
"A vcluster instance created via vcluster provider is configured using clusterctl parameters and environment variables: 
- `CHART_NAME` (required/optional) (type): description
- `CHART_REPO` (required/optional) (type): description
- `CHART_VERSION` (required/optional) (type): description
- `VCLUSTER_HOST` (required/optional) (type): description
- `VCLUSTER_PORT` (required/optional) (type): description
- `HELM_VALUES` (required/optional) (type): description"